problemi/boltzmannDistribution/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+find_package(ROOT REQUIRED COMPONENTS Hist Graf Graf3d Gpad)
+
+add_subdirectory(src)
+
+add_executable(boltzmannDistribution.out boltzmannDistribution.cpp)
```

Comment:
nomi dei file minuscoli (come gia' detto in altri commenti, non lo ripeto piu' :-))